storage/innobase/xtrabackup/src/xtrabackup.cc

```diff
@@ -2541,6 +2540,45 @@ static bool innodb_end(void) {
   return (FALSE);
 }
 
+/* Read backup meta info.
+@param[in] filename filename to read mysql version
+@return TRUE on success, FALSE on failure. */
+static bool xtrabackup_read_info(char *filename) {
+  FILE *fp;
+  bool r = TRUE;
+  char mysql_server_version_str[30] = ""; /* 8.0.20.debug| 8.0.20 */
+  unsigned long mysql_server_version;
+
+  fp = fopen(filename, "r");
+  if (!fp) {
+    msg("xtrabackup: Error: cannot open %s\n", filename);
+    return (FALSE);
+  }
+  /* skip uuid, name, tool_name, tool_command, tool_version, ibbackup_version */
+  for (int i = 0; i < 6; i++) {
+    char c;
+    do {
+      c = fgetc(fp);
+    } while (c != '\n');
+  }
+
+  if (fscanf(fp, "server_version = %29s\n", mysql_server_version_str) != 1) {
+    r = FALSE;
+    goto end;
+  }
+  mysql_server_version = get_version_number(mysql_server_version_str);
+
+  ut_ad(mysql_server_version > 80000 && mysql_server_version < 90000);
+  if (mysql_server_version < 80023) {
+    cfg_version = IB_EXPORT_CFG_VERSION_V4;
+  } else if (mysql_server_version < 80026) {
+    cfg_version = IB_EXPORT_CFG_VERSION_V5;
+  }
```

Comment:
as safety check I have added new test case xb_export_check_cfg.sh which checks what is version of CFG. if version is greater than 6 . it will fail so we will come to know.

Reply:
with PXB 8.0.27. we will have CFG 6.